javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -235,19 +235,21 @@ public static boolean areSameIgnoringValues(AnnotationMirror a1, AnnotationMirro
     }
 
     /** Checks that the annotation {@code am} has the name {@code aname}. Values are ignored. */
-    public static boolean areSameByName(AnnotationMirror am, /*@Interned*/ String aname) {
+    public static boolean hasName(AnnotationMirror am, /*@Interned*/ String aname) {
         // Both strings are interned.
         return annotationName(am) == aname;
     }
 
     /** Checks that the annotation {@code am} has the name of {@code anno}. Values are ignored. */
-    public static boolean areSameByClass(AnnotationMirror am, Class<? extends Annotation> anno) {
-        /*@Interned*/ String canonicalName = annotationClassNames.get(anno);
-        if (canonicalName == null) {
+    public static boolean hasClass(AnnotationMirror am, Class<? extends Annotation> anno) {
+        /*@Interned*/ String canonicalName;
+        if (annotationClassNames.containsKey(anno)) {
```

Comment:
The method names areSameByName and areSameByClass are similar to areSameIgnoringValues, but the signature is different, and this makes client code confusing when it uses both areSameIgnoringValues and one of the other two.

The method "areSameByName" isn't comparing two things that are the same by name.
If you have better ideas for the names, let us know, but the current ones seem misleading.